docker-compose.yml

```diff
@@ -194,7 +192,7 @@ services:
      - ./validator-ejector:/exitmessages
    restart: unless-stopped
    environment:
-     -  EXECUTION_NODE=${VE_EXECUTION_NODE_URL:-http://geth:8545}
+     -  EXECUTION_NODE=${VE_EXECUTION_NODE_URL:-http://nethermind:8545}
```

Comment:
do we need to change `ORACLE_ADDRESSES_ALLOWLIST`?

Reply:
Hmm this is a Lido detail, not really sure, but I believe yes.

You can find them on the validator-ejector github page.